dimagi/utils/post.py

```diff
@@ -64,6 +65,22 @@ def simple_post(data, url, content_type="text/xml", timeout=60, headers=None, au
     return requests.post(url, data, **kwargs)
 
 
+def simple_xml_post(data, url, operation):
+    client = Client(url)
```

Comment:
did you look into the caching of this? I think this will do multiple requests to the Nikshay server every time we send them a message otherwise.